docs/about/release-notes.rst

```diff
@@ -12,6 +12,9 @@ Features
 * New ``profiler`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1D and 2D projections only).
 * Sliders have a thickness slider associated with them and can be used to show slices of arbitrary thickness.
 * Can hide/show individual masks on plots.
+* Add binned data support, replacing "event list" dtypes as well as "realign" support.
```

Comment:
Can we internally link to relevant pages/notebooks?

Reply:
We could, but since those are changing over time the links would go dead sooner or later and we would need to refactor our relase note pages.